op-node/chaincfg/chains_test.go

```diff
@@ -89,22 +89,22 @@ var sepoliaCfg = rollup.Config{
 			GasLimit:    30000000,
 		},
 	},
-	BlockTime:              2,
-	MaxSequencerDrift:      600,
-	SeqWindowSize:          3600,
-	ChannelTimeoutBedrock:  300,
-	L1ChainID:              big.NewInt(11155111),
-	L2ChainID:              big.NewInt(11155420),
-	BatchInboxAddress:      common.HexToAddress("0xff00000000000000000000000000000011155420"),
-	DepositContractAddress: common.HexToAddress("0x16fc5058f25648194471939df75cf27a2fdc48bc"),
-	L1SystemConfigAddress:  common.HexToAddress("0x034edd2a225f7f429a63e0f1d2084b9e0a93b538"),
-	RegolithTime:           u64Ptr(0),
-	CanyonTime:             u64Ptr(1699981200),
-	DeltaTime:              u64Ptr(1703203200),
-	EcotoneTime:            u64Ptr(1708534800),
-	FjordTime:              u64Ptr(1716998400),
-	GraniteTime:            u64Ptr(1723478400),
-	// HoloceneTime: TBD
+	BlockTime:               2,
+	MaxSequencerDrift:       600,
+	SeqWindowSize:           3600,
+	ChannelTimeoutBedrock:   300,
+	L1ChainID:               big.NewInt(11155111),
+	L2ChainID:               big.NewInt(11155420),
+	BatchInboxAddress:       common.HexToAddress("0xff00000000000000000000000000000011155420"),
+	DepositContractAddress:  common.HexToAddress("0x16fc5058f25648194471939df75cf27a2fdc48bc"),
+	L1SystemConfigAddress:   common.HexToAddress("0x034edd2a225f7f429a63e0f1d2084b9e0a93b538"),
+	RegolithTime:            u64Ptr(0),
+	CanyonTime:              u64Ptr(1699981200),
+	DeltaTime:               u64Ptr(1703203200),
+	EcotoneTime:             u64Ptr(1708534800),
+	FjordTime:               u64Ptr(1716998400),
+	GraniteTime:             u64Ptr(1723478400),
+	HoloceneTime:            u64Ptr(1732633200),
```

Comment:
UTC
2024-11-26 15:00:00

Reply:
yep that matches, put in the reference SCR PR and excerpt in the description.